projects/ibmswtpm2/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER john.s.andersen@intel.com
+ARG SIM_DL_URL=https://astuteinternet.dl.sourceforge.net/project/ibmswtpm2/ibmtpm1332.tar.gz
```

Comment:
I sometimes see sourceforge mirrors go down (but I'm not sure downloads.sourceforge.net is more reliable either).